spock-specs/src/test/groovy/org/spockframework/smoke/condition/ExceptionsInConditions.groovy

```diff
@@ -0,0 +1,136 @@
+package org.spockframework.smoke.condition
+
+import junit.framework.Assert
+import org.spockframework.EmbeddedSpecification
+import org.spockframework.runtime.ConditionNotSatisfiedError
+import org.spockframework.runtime.SpockException
+import spock.lang.FailsWith
+import spock.lang.Specification
+
+
+class ExceptionsInConditions extends EmbeddedSpecification {
+  def "null pointer exception in condition"() {
+    when:
+    runner.runFeatureBody(
+        """
```

Comment:
Please fix the indenting (starting and closing `"""` should be on the same level) and use `given` or `setup` in the feature bodies.